float64[] posture
float64[] posture_prob
float64[] position_x
float64[] position_y
float64[] centroid_x
float64[] centroid_y
float64[] distance
float64[] orientation
float64[] sensor
float64[] sensor_t0
float64[] sensor_t1
float64[] sensor_c0
float64[] sensor_c1
float64[] motion
float64[] area

